Update "Editing markdown in the Visual Editor" (#367)
CloudCannon/platform-documentation: articles/editing-markdown-files-in-the-visual-editor.mdx

  - introducing-the-visual-editor
related_links:
---

> When opened in the Visual Editor, Markdown files can be edited in-place using a Content Editor embedded on your page.

Reviewer: ```suggestion
You can edit markdown files in the Visual Editor using an in-place Rich Text editor embedded on your page.
```
